driver/apdu/pcsc.c

```diff
@@ -25,6 +26,11 @@
 #define APDU_CLOSELOGICCHANNEL "\x00\x70\x80\xFF\x00"
 #define APDU_SELECT_HEADER "\x00\xA4\x04\x00\xFF"
 
+static char *BLOCKLIST[] = {
```

Comment:
May be better to introduce an environment variable like https://github.com/martinpaljak/GlobalPlatformPro/wiki/Getting-started#gp_reader_ignore